cg/constants/slurm.py

```diff
@@ -1,6 +1,24 @@
 from enum import Enum
 
+from cg.utils.enums import StrEnum
+
 
 class Slurm(Enum):
     PARTITION: str = "partition"
     MAX_NODE_MEMORY: int = 180
+
+
+class ContextEnv:
```

Comment:
Should ContextEnv and SlurmAccount be enums as well?

Reply:
This was removed following Vadym's comment. That logic is now in the config file and this is unnecessary.